File: kiki616/ABC/B/010.py
n = int(input())
a = list(map(int,input().split()))
answer = 0
for i in a:
    cnt = 0
    while True:
        if (i - cnt)%2 == 1 and not (i - cnt)%3 == 2:
            answer = answer + cnt
            break
        else:
            cnt = cnt + 1
print(answer)

File: OCH_Tao/ABC/B/010.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(input[0], 10);
const A = (input[1].split(" ")).map(x => parseInt(x, 10));
let cnt = 0;
for (let i = 0; i < N; i++) {
  let x = A[i];
  while (x % 2 === 0 || x % 3 === 2) {
    cnt++;
    x--;
  }
}
console.log(cnt);